implementation/src/main/java/io/smallrye/config/SmallRyeConfigSourceInterceptorContext.java

```diff
@@ -15,4 +17,14 @@
     public ConfigValue proceed(final String name) {
         return interceptor.getValue(next, name);
     }
+
+    @Override
+    public Iterator<String> names() {
```

Comment:
The `proceed` method is a verb.  Maybe this should be `iterateNames` so that it is also a verb?